### PR TITLE
Prysm BN/VC: Add `tty: true`

### PR DESCRIPTION
**Pull request by nalepae**
It will enable pretty logging for Prysm BN/VC.

Before:
image

After:
image

### CODE REVIEW

**Review by barnabasbusa — APPROVED** [review]

**Comment by barnabasbusa** [comment]
Could you please make it configurable, instead of hardcoding it to true?

**Comment by nalepae** [comment]
> Could you please make it configurable, instead of hardcoding it to true?

Fixed in 0c8da8f9b459d3d2db1d6ef846c787e918f123a7

File: roles/prysm/defaults/main.yaml [hunk 1]
```diff
@@ -49,6 +49,7 @@ prysm_container_volumes:
 prysm_container_security_opts: []
 prysm_container_stop_timeout: "300"
 prysm_container_networks: []
+prysm_container_tty: true
```
Comment (barnabasbusa) [inline roles/prysm/defaults/main.yaml:52]:
```suggestion
prysm_container_tty: false
```

File: roles/prysm/defaults/main.yaml [hunk 2]
```diff
@@ -101,6 +102,7 @@ prysm_validator_container_volumes:
 prysm_validator_container_security_opts: []
 prysm_validator_container_stop_timeout: "300"
 prysm_validator_container_networks: []
+prysm_validator_container_tty: true
```
Comment (barnabasbusa) [inline roles/prysm/defaults/main.yaml:105]:
```suggestion
prysm_validator_container_tty: false
```

**Pull request merged**